app/templates/search/catalogue/accord.html

```diff
@@ -0,0 +1,308 @@
+                <div class="tna-form__group">
+                  <fieldset class="tna-form__fieldset">
+                    <legend class="tna-form__legend">
+                      For example, 27 3 2007 or 2007<br>
+                      From:
+                    </legend>
+                    <div class="tna-date-input" id="date">
+                      <div class="tna-date-input__item">
+                        <label for="date-day" class="tna-date-input__item-label">
+                          Day
+                        </label>
+                        <input type="text" id="date-day" value="" name="date-day" class="tna-date-input__item-input" inputmode="numeric">
+                      </div>
+                      <div class="tna-date-input__item">
+                        <label for="date-month" class="tna-date-input__item-label">
+                          Month
+                        </label>
+                        <input type="text" id="date-month" value="" name="date-month" class="tna-date-input__item-input" inputmode="numeric">
+                      </div>
+                      <div class="tna-date-input__item">
+                        <label for="date-year" class="tna-date-input__item-label">
+                          Year
+                        </label>
+                        <input type="text" id="date-year" value="" name="date-year" class="tna-date-input__item-input tna-date-input__item-input--wider" inputmode="numeric">
+                      </div>
+                    </div>
+                    <legend class="tna-form__legend">
+                      To:
+                    </legend>
+                    <div class="tna-date-input" id="date">
+                      <div class="tna-date-input__item">
+                        <label for="date-day" class="tna-date-input__item-label">
+                          Day
+                        </label>
+                        <input type="text" id="date-day" value="" name="date-day" class="tna-date-input__item-input" inputmode="numeric">
+                      </div>
+                      <div class="tna-date-input__item">
+                        <label for="date-month" class="tna-date-input__item-label">
+                          Month
+                        </label>
+                        <input type="text" id="date-month" value="" name="date-month" class="tna-date-input__item-input" inputmode="numeric">
+                      </div>
+                      <div class="tna-date-input__item">
+                        <label for="date-year" class="tna-date-input__item-label">
+                          Year
+                        </label>
+                        <input type="text" id="date-year" value="" name="date-year" class="tna-date-input__item-input tna-date-input__item-input--wider" inputmode="numeric">
+                      </div>
+                    </div>
+                  </fieldset>
+                </div>
```

Comment:
This might be easier to maintain if we used two date input components: https://nationalarchives.github.io/design-system/components/date-input/

I still feel that it would be easier for users to use the date search component (https://nationalarchives.github.io/design-system/components/date-search/) but I think that's something that needs to be user tested. Maybe out of scope for this review.

Reply:
I'm sticking withe the html layout for the moment as when I try to implement via nunjuscks, I'm getting the following error: `{{ no such element: dict object['label'] }}`

I have imported the date input component:
`{% from "components/date-input/macro.html" import tnaDateInput %}`

I'll flag to Simon that the date search component is also available now.

Reply:
The date range elements now handled via nunjucks component